actors/builtin/network.go

```diff
@@ -2,22 +2,27 @@ package builtin
 
 import "fmt"
 
+// PARAM_SPEC
 // The duration of a chain epoch.
-// This is used for deriving epoch-denominated periods that are more naturally expressed in clock time.
+// Motivation: It guarantees that a block is propagated and WinningPoSt can be successfully done in time all supported miners.
+// Usage: It is used for deriving epoch-denominated periods that are more naturally expressed in clock time.
 // TODO: In lieu of a real configuration mechanism for this value, we'd like to make it a var so that implementations
 // can override it at runtime. Doing so requires changing all the static references to it in this repo to go through
 // late-binding function calls, or they'll see the "wrong" value.
 // https://github.com/filecoin-project/specs-actors/issues/353
 // If EpochDurationSeconds is changed, update `lambda` and `expLamSubOne` in ./reward/reward_logic.go
-const EpochDurationSeconds = 25
+const EpochDurationSeconds = 30
```

Comment:
You can't change this without doing what the comment above talks about. Please revert this. We can track changing this in a separate issue. I have not yet received a concrete request for that change.

Reply:
Will be tracked here: https://github.com/filecoin-project/lotus/issues/2547

Reply:
Confirmation from @whyrusleeping here: https://github.com/filecoin-project/lotus/issues/2547#issuecomment-663127215